gluten-core/src/main/scala/io/glutenproject/expression/ExpressionConverter.scala

```diff
@@ -26,7 +26,7 @@ import org.apache.spark.sql.catalyst.optimizer.NormalizeNaNAndZero
 import org.apache.spark.sql.execution._
 import org.apache.spark.sql.execution.exchange.BroadcastExchangeExec
 
-object ExpressionConverter extends Logging {
+case class ExpressionConverter(validation: Boolean) extends Logging {
```

Comment:
Can we add some notes here for parameter `validation`? By looking into ExpressionConverter's code one doesn't realize where validation is happened.

Reply:
add a comment here

Reply:
Thanks!